### DISCUSSION TITLE
Welcome to WiCAN Discussions!

### DISCUSSION THREAD

**Discussion by meatpiHQ** [meatpiHQ/wican-fw#22] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by J85FR** [comment]
hello,
I'm trying to connect to a Wican USB with busmaster but I don't know what is the trouble.
the connection fails with the message 'Failed to start the hardware'.
the green led goes on, then off after that message, which comes immediately.
I'm sure that the ip:port are good, when these ar wrong the led does not goes on and the message occurs later.
I use the slcan protocol, the device connects properly with a browser (firefox).

I use 192.168.1.11:3334, this is the ip my internet box has assigned to the module over wifi.
I have reinstalled busmaster.
it seems that some dll are missing, with DEPEND.EXE application, I have found this:
![image](https://github.com/meatpiHQ/wican-fw/assets/141920821/1cee19f1-af96-4d86-be27-d383b47b9891)
the pc runs windows7 pro 64 bit, the vcredist are 32bit, and I have on the machine the following:
- 2010
- 2013 (included in busmaster instal package)
I tried to install vcredist 2015 and 2017, but the issue is the same.
what can I do ?
is there another application I could use to check the hardware ?
thanks in advance

**Reply by meatpiHQ** [reply]
@J85FR Can you please post a screenshot of you configuration webpage? 

Did you download BUSMaster from this link? https://drive.google.com/drive/folders/1qJelUAHGrn_YbNIP0Jk_KmNENG-hKbtl

**Reply by J85FR** [reply]
yes:
![wican usb config](https://github.com/meatpiHQ/wican-fw/assets/141920821/488d4ebb-8ece-4a06-9237-b7ee825b6257)
![wican usb about](https://github.com/meatpiHQ/wican-fw/assets/141920821/76b4409a-69c6-4dfa-9bc5-09bdccb8c469)

regarding the download link, I have downloaded several versions, but I think I have not installed this one, I'm going to try it.

**Reply by J85FR** [reply]
it's the same problem with this version
![image](https://github.com/meatpiHQ/wican-fw/assets/141920821/6acbdd2e-6e1a-4ae4-9198-dbfb5896c58f)
![image](https://github.com/meatpiHQ/wican-fw/assets/141920821/9e34f404-5699-4a86-b155-65a83c131644)

**Reply by meatpiHQ** [reply]
@J85FR please update your device to the latest firmware version

https://github.com/meatpiHQ/wican-fw/releases

**Reply by J85FR** [reply]
following the documentation, I have several questions regarding esp32 flashing tool:

- I suppose that I must shorten both pins in the red frame on the image below, the document corresponds to the OBD2 module, for the USB module, connector position is not at the same place
![image](https://github.com/meatpiHQ/wican-fw/assets/141920821/cbe477d3-4cde-4534-aef9-be5a01f4ff69)
- is it necessary to click on the CombineBin button, the file selection window does not provide any button to validate and it is not possible to enter a file name... double clicking tries to launch the selected file
![image](https://github.com/meatpiHQ/wican-fw/assets/141920821/8a49f13b-2e7a-4e4a-a196-bbcf98c18050)
- in case I update from the web page, should I flash the 4 files successively, and in which order ? or only the first one ?

**Reply by meatpiHQ** [reply]
@J85FR I suggest you use the webpage to update. You only need one file, wican-fw-isb_v180_hv150.bin

If you're using the USB, no need to combine files.

**Reply by J85FR** [reply]
I have updated through the webpage, and busmaster now works fine!
device is connected, and I have no more start error.
![image](https://github.com/meatpiHQ/wican-fw/assets/141920821/45ff28fa-f36a-4cb6-b7d2-c32bb52036ca)
thank you for your help
